TestApp/ios/Podfile.lock

```diff
@@ -372,7 +377,7 @@ SPEC CHECKSUMS:
   appcenter-core: 98c617ae028c90020e404f020c37d7cb189adf4c
   appcenter-crashes: add23a3c3cf0403e1f54c39d85e9b3c5db7279e7
   appcenter-push: 9a88b078ad61718947c92561b5b86da54e185582
-  AppCenterReactNativeShared: c57d2566896ba19a6193046c8966691c41b4be93
+  AppCenterReactNativeShared: d030cbd4c11902dcff4a9cea80dac66714cc2df6
```

Comment:
Is this required?

Reply:
@annakocheshkova @russelarms 
Checksum changes every time your build xcode project, which means, unfortunately, it will change every time we run `TestApp/update-npm-package.sh` during normal development flow even there is no change regarding iOS.
The checksum of `AppCenterReactNativeShared` is not providing much value to us, but I am fine to live with it despite of the annoyance.